code/modules/mod/mod_theme.dm

```diff
@@ -392,6 +392,16 @@
 	slowdown_inactive = 0.5
 	slowdown_active = 0
 	allowed_suit_storage = list(
+		/obj/item/rpd,
+		/obj/item/rcs,
+		/obj/item/storage/bag/mail,
+		/obj/item/envelope,
+		/obj/item/stamp,
+		/obj/item/pen,
+		/obj/item/paper,
+		/obj/item/mail_scanner,
+		/obj/item/eftpos,
+		/obj/item/melee/classic_baton/telescopic,
 	)
```

Comment:
```suggestion
		/obj/item/melee/classic_baton/telescopic
	)
```

Reply:
QOL being removed to ensure the fix is merged before commit updates in the coming round.